Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -898,14 +902,38 @@ def get_last_run():
     return last_run
 
 
+def parse_fetch_time_to_minutes():
+    """
+    Calculate the time to fetch back in minutes
+    Returns (int): Time to fetch back in minutes
+    """
+    number_of_times, time_unit = FETCH_TIME.split(' ')
+    if str(number_of_times).isdigit():
+        number_of_times = int(number_of_times)
+    else:
+        return_error("Error: Invalid fetch time, need to be a positive integer with the time unit afterwards"
```

Comment:
we should return_error only in main because it does sys.exit. Instead prefer to raise exceptions.